src/enums/element_sub_type.py

```diff
@@ -2,19 +2,35 @@
 
 
 class ElementSubType(Enum):
+    # BLOCKS
     OVERWORLD_BLOCK = "OVERWORLD_BLOCK"
     UNDERGROUND_BLOCK = "UNDERGROUND_BLOCK"
+    OVERWORLD_BRICK = "OVERWORLD_BRICK"
+    OVERWORLD_BROKE_BRICK = "OVERWORLD_BROKE_BRICK"
+
     SMALL_BUSH = "SMALL_BUSH"
     MEDIUM_BUSH = "MEDIUM_BUSH"
     BIG_BUSH = "BIG_BUSH"
     DEFAULT_CASTLE = "DEFAULT_CASTLE"
+
+    # CLOUDS
     SMALL_CLOUD = "SMALL_CLOUD"
     MEDIUM_CLOUD = "MEDIUM_CLOUD"
     BIG_CLOUD = "BIG_CLOUD"
-    DEFAULT_COIN = "DEFAULT_COIN"
+
     DEFAULT_FLAG = "DEFAULT_FLAG"
-    DEFAULT_MISTERY_BOX = "DEFAULT_MISTERY_BOX"
+
     DEFAULT_MOUNTAIN = "DEFAULT_MOUNTAIN"
-    SMALL_PIPE = "SMALL_PIPE"
-    MEDIUM_PIPE = "MEDIUM_PIPE"
-    BIG_PIPE = "BIG_PIPE"
+
+    # PIPES
+    # Head
+    PIPE_HEAD = "PIPE_HEAD"
+
+    # Body
+    PIPE_BODY = "PIPE_BODY"
```

Comment:
When we are going to draw different heights of pipes, how we achieve that with this constant?

Reply:
Eso dependeria de la altura de las tuberias segun su posicion, ya que la idea seria que el PIPE_HEAD seria por donde el PLAYER entraria a la tuberia, luego el PIPE_BODY seria simplemente para darle la altura a la tuberia para evitar que sean filas de solo los PIPE_HEAD